index.html

```diff
@@ -1276,11 +1221,11 @@ <h4>
         See the [[[#writing-or-pushing-content]]] section.
       </p>
       <p>
-        Pushing <a>Web NFC content</a> to an <a>NFC tag</a> does not need to
-        <a>obtain permission</a>, if the <a>message identifier host</a> of the
-        <a>author type record</a> on that <a>NFC tag</a> is equal to the
-        <a data-cite="dom#concept-host-serializer">serialized host</a>
-        of the <a>current settings object</a>'s origin.
+        Pushing an <a>NDEF message</a> to an <a>NFC tag</a> does not need to
+        <a>obtain permission</a>, if the existing <a>NDEF message</a> only
+        contains <a>NDEF record</a>s without <a>ID field</a>s, or with <a>
+        ID field</a>s matching the [=host/registrable domain=] of the
+        <a>current settings object</a>'s origin.
```

Comment:
A tricky sentence from security pov. We'd want to encourage using the ID, not the contrary :).
I would require permission when writing a message in which the IDs per record (payload id's) are not specified.
To be security-reviewed?

Reply:
I am totally OK with changing this. Please file issue and we take it from there

Reply:
Added a note in the spec